vumi/components/tests/test_message_store.py

```diff
@@ -1249,6 +1250,12 @@ def test_reconcile_cache(self):
         self.assertFalse((yield self.store.needs_reconciliation(batch_id)))
         self.assertFalse(
             (yield self.store.needs_reconciliation(batch_id, delta=0)))
+
+        inbound_uniques = yield self.store.cache.count_from_addrs(batch_id)
+        self.assertEqual(inbound_uniques, 1)
+        outbound_uniques = yield self.store.cache.count_to_addrs(batch_id)
+        self.assertEqual(outbound_uniques, 1)
```

Comment:
These tests should probably be updated to use more than one address, and ideally different numbers of addresses (and messages, really) between inbound and outbound to make sure we're updating the correct keys with the correct values.